modules/ROOT/pages/sap/sap-connector-xml-maven.adoc

```diff
@@ -0,0 +1,105 @@
+= SAP XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule application in XML either from the Studio XML editor 
+or from a text editor, perform the following actions to get access to the connector:
+
+. Add the namespace for the connector to your XML flow.
+. Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the 
+canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
```

Comment:
Does "your configuration XML" equate to the aforementioned "Studio XML editor or text editor"?
GLOBALLY attempt to use consistent terms throughout a topic, one term for one thing.